Adyen/UI/List/ListItemView.swift

```diff
@@ -161,13 +184,4 @@ public final class ListItemView: UIView, AnyFormItemView {
         imageView.layer.borderColor = item?.style.image.borderColor?.cgColor ?? UIColor.Adyen.componentSeparator.cgColor
     }
     
-    override public var intrinsicContentSize: CGSize {
```

Comment:
This weird was probably here to fix something in iOS 10 or 11..

Reply:
I'll test it and see

Reply:
works fine on iOS 11, 12, 13 and 14